Resources/Textures/Objects/Specific/Hydroponics/cherry.rsi/meta.json

```diff
@@ -0,0 +1,44 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Made by RumiTiger",
```

Comment:
Since this sprite is edited from orange trees (or something adjacent), you need to fix the copyright to reflect that by including the original source.